internal/providers/terraform/aws/apigatewayv2_api.go

```diff
@@ -0,0 +1,209 @@
+package aws
+
+import (
+	"github.com/infracost/infracost/internal/schema"
+	"github.com/shopspring/decimal"
+)
+
+func GetApiGatewayv2ApiRegistryItem() *schema.RegistryItem {
+	return &schema.RegistryItem{
+		Name:  "aws_apigatewayv2_api",
+		Notes: []string{
+			"WebSocket Connection minutes is not yet supported",
```

Comment:
@alikhajeh1 do you want me to add the cost component in anyway? The reason I left it out was because of the calculation variation on websocket connections. It can be varied per second / minute / hour & the number of connections rate can be per second / minute / hour / day.

Reply:
@robh007 I think it's helpful to see that, even if there is no infracost provider support for it, e.g.:
```
  └─ Connection minutes          -  1M minutes     0.025       -          -
```
If we add the provider support, would having `average_connection_duration` (in minutes) and `monthly_connections` help? so the user inputs those and we multiply them to calculate the total connection minutes.